### PR TITLE
fix non-existent-field issue

### PR DESCRIPTION
**Pull request by GuyAfik**
## Description
fixed a bug that if id_set_file was not provided the mapper format would fail.

### CODE REVIEW

File: demisto_sdk/commands/format/update_mapper.py [hunk 1]
```diff
@@ -67,6 +67,9 @@ def remove_non_existent_fields(self):
         """
         Remove non-existent fields from a mapper.
         """
+        if not self.id_set_file:
```
Comment (dorschw) [inline demisto_sdk/commands/format/update_mapper.py:70]: please add logging, explaining what happens what the behavior is with and id_set, and without it

**Review by JudahSchwartz — APPROVED** [review]
@dorschw can this be merged? Does it really need a changelog entry?

File: demisto_sdk/commands/format/update_layout.py [hunk 2]
```diff
@@ -267,6 +267,7 @@ def remove_non_existent_fields_container_layout(self):
         Remove non-existent fields from a container layout.
         """
         if not self.id_set_file:
+            click.secho('Skipping non-existent-fields formatting as id_set_path argument is missing', fg='yellow')
```
[the remark this answers is not in the document]
Comment (GuyAfik) [reply demisto_sdk/commands/format/update_layout.py:270]: ill add the name of the file that's a good point

File: demisto_sdk/commands/format/update_layout.py [hunk 3]
```diff
@@ -267,6 +267,10 @@ def remove_non_existent_fields_container_layout(self):
         Remove non-existent fields from a container layout.
         """
         if not self.id_set_file:
+            click.secho(
```
Comment (dorschw) [inline demisto_sdk/commands/format/update_layout.py:270]: please replace with `logger = logging.getLogger('demisto-sdk')`, and `logger.warning`

**Pull request merged**